### PR TITLE
Setup Github actions

### LINKED ISSUE #6: Minor: create tag any time there is a new sharp layer version
I follow this on my feed. Creating tags will notify me when there's a new version available.

Thank you!

### PR DESCRIPTION
**Pull request by hansottowirtz**
This PR adds files to automatically build the layer zip and publish it to Github in a release  (if there's a new version of sharp)

It will look like this: https://github.com/bubblydoo/lambda-layer-sharp/releases

I also removed minimization because it makes debugging much harder. The difference is 159KB.

Closes #6.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = {
     })
   ],
   optimization: {
-    minimize: true,
+    minimize: false,
```
Comment (o-alexandrov) [inline webpack.config.js:38]: Could you please share your view on why you think minimization should be disabled?

Comment (hansottowirtz) [reply webpack.config.js:38]: I forgot to mention this. I encountered an error once and the minimization made it very hard to debug. The difference is only 156KB, I think that is not enough to warrant difficult and likely debugging.

Comment (Umkus) [reply webpack.config.js:38]: Probably defaulting to `true`, but optionally checking `process.env.NODE_ENV` (which should be set to `true` in the GH actions env) would be best.

File: README.md [hunk 2]
```diff
@@ -2,12 +2,8 @@
 This AWS lambda layer contains a pre-built [sharp](https://www.npmjs.com/package/sharp) npm library.
 It is optimized for the most frugal space usage possible.
 
-|Sharp version| Layer size|
-|---|---|
-|[0.26.0](https://github.com/lovell/sharp/releases/tag/v0.26.0)|9.68MB|
-
 # Getting
-A pre-built layer zip file is available at [`dist/sharp-layer.zip`](./dist/sharp-layer.zip).
+A pre-built layer zip file is available on the [Releases page](https://github.com/bubblydoo/lambda-layer-sharp/releases), alongside the size of the layer.
```
Comment (Umkus) [inline README.md:6]: Absolute links are not portable.

**Pull request merged**